thor/tests/memory/test_memory.py

```diff
@@ -0,0 +1,350 @@
+"""
+A series of unit tests to run with memray for memory profiling.
+
+# These test should be run with the following command:
+pytest -v --memray --native --memray-bin-path=.cache/bench/memory . -m memory
+
+# You can then generate flamegraphs and other reports with something like this:
+memray flamegraph .cache/bench/memory/path_to_output.bin
+
+# Open in a browser
+open .cache/bench/memory/path_to_output.html
+
+# You can also view a graph of the _system_ memory usage
+# for the duration of the test
+open .cache.bench/memory/[session_name]/[test_name].png
+
+"""
+import os
+import subprocess
+import threading
+import time
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import psutil
+import pytest
+
+TEST_ORBIT_ID = "896831"
+# FIXTURES_DIR = Path(__file__).parent / "fixtures"
+FIXTURES_DIR = Path("/opt/volumes/")
+CONFIG_PROCESSES = [1, 4]
```

Comment:
I ran these instructions using the commented "FIXTURES_DIR" and they worked in my conda environment. I didn't try the Docker-oriented route.